lib/season_statable.rb

```diff
@@ -19,20 +19,20 @@ def wins_per_coach(season)
     coach_wins
   end
 
-  def winningest_coach(season)
-    coach_win_percentage = wins_per_coach(season).transform_values do |value| 
+  def coach_sucess_percentage(season)
```

Comment:
yeah very nice here, this was also used a couple times, so this def drys up the code a bit more.